data/init-cli/docker_init.yaml

```diff
@@ -65,6 +67,7 @@ examples: |-
     - .dockerignore
     - Dockerfile
     - compose.yaml
+    - README.Docker.md
   
   Let's get started!
```

Comment:
should php also be added to the list of templates below in the sample cli output on line 74?